docs/docs-content/getting-started/getting-started.md

```diff
@@ -2,35 +2,95 @@
 sidebar_label: "Getting Started"
 title: "Getting Started"
 description: "Spectro Cloud Getting Started"
-hide_table_of_contents: false
+hide_table_of_contents: true
 sidebar_custom_props:
   icon: "overview"
 tags: ["getting-started"]
 ---
 
-This page gives an overview of how to get started with Spectro Cloud Palette and begin leveraging its Kubernetes
+This page gives you an overview of how to get started with Spectro Cloud Palette and begin leveraging its Kubernetes
 full-stack management at scale. Palette's unique capabilities provide end-to-end declarative cluster management, cluster
-monitoring and reconciliation, as well as adherence to the highest security standards. It provides support for public
-cloud providers, data centers, bare metal and edge, so you can use Palette no matter the complexity of your production
-environments.
+monitoring and reconciliation, as well as enterprise-grade security.
+
+Check out the following short video for an overview of some the key functionality that Palette provides. Spectro Cloud
+Palette provides developers and platform engineers with complete control of their Kubernetes clusters. It provides
+support for public cloud providers, data centers, bare metal and edge, so you can use Palette no matter the complexity
+or location of your production environments.
```

Comment:
```suggestion
Palette provides developers and platform engineers with complete control of their Kubernetes clusters. The platform provides support for public cloud providers, data centers, bare metal, and edge so that you can deploy and use Kubernetes no matter the complexity or location of your production environments. Check out the following short video for a quick overview of Palette's key functionalities.
```

Changed the order so that when we state "Check out the following video" the video is actually the next item :)